### DISCUSSION TITLE
AddMedia without upload.

### DISCUSSION THREAD

**Discussion by masitings** [spatie/laravel-medialibrary#2945] [Q&A] [answered]
Hi there, 

is there any way to upload the file manually and then add media using the url without upload again ? I've try using addMediaFromUrl but it needs to upload the file again.

**Comment by PaolaRuby** [comment]
Yes, read the documentation

**Reply by masitings** [reply]
Could you give me the example of code ? I've read the documentation but it leads me still do the upload. Much appriciate..

**Comment by systemsolutionweb** [comment, marked as answer]
In the documentation you will find what you are looking for
